File: benchmark/results/java-multilang/run-001/b-baseline/output/com/metaengine/demo/notification/services/NotificationService.java
package com.metaengine.demo.notification.services;

import java.util.List;

import com.metaengine.demo.notification.aggregates.Notification;

/** NotificationService for the notification domain. */
public class NotificationService {

    /** Creates a new Notification from the partial input. */
    public Notification create(Notification input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Finds a Notification by id. May return null. */
    public Notification findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Lists Notifications up to the given limit. */
    public List<Notification> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Deletes the Notification with the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/ts-monolith-opus-script/run-002/b-baseline/output/src/notifications/services/NotificationService.ts
import { EmailTemplate } from "../value-objects/EmailTemplate";
import { Id } from "../../shared/value-objects/Id";

/** CRUD operations for notification templates. */
export class NotificationService {
  create(input: Partial<EmailTemplate>): EmailTemplate {
    void input;
    throw new Error("not implemented");
  }

  findById(id: Id): EmailTemplate | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): EmailTemplate[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: Id): void {
    void id;
    throw new Error("not implemented");
  }
}
